### ISSUE TITLE
Price validation message breaks UI

### ISSUE THREAD

**Issue by amdrew** [SellBirdHQ/theme#237]
By default the Parsley validation script adds its validation message directly after an input field. Because of how https://github.com/SellBirdHQ/theme/issues/134 works, the validation now breaks the layout (uses flexbox). To fix, we'll need to move the validation outside of the `.input-group` div.

Before:

<img width="330" alt="screen shot 2018-05-28 at 10 48 45 am" src="https://user-images.githubusercontent.com/52581/40591425-474ef928-6265-11e8-80af-cca2c401d45c.png">

Aim:

<img width="325" alt="screen shot 2018-05-28 at 10 49 10 am" src="https://user-images.githubusercontent.com/52581/40591429-4e9c2156-6265-11e8-9d11-accce94ebd8f.png">

**Comment by amdrew** [comment]
PR: https://github.com/SellBirdHQ/theme/pull/238

**Comment by amdrew** [comment]
This still needs a bit of work. Since the fields repeat, you can add 3 pricing fields and all the errors will show in only one of the error divs. Assigning over to you @DrewAPicture to work your magic. Each field will need its own unique error div from the look of things.

<img width="816" alt="screen shot 2018-05-28 at 11 27 23 am" src="https://user-images.githubusercontent.com/52581/40591724-670abac2-626a-11e8-8669-b59da611a86d.png">

**Comment by amdrew** [comment]
Adjusted the markup to fix the rounded corners on the prepended `$`. Before:
<img width="319" alt="screen shot 2018-05-28 at 11 34 51 am" src="https://user-images.githubusercontent.com/52581/40591783-7e8b2f0a-626b-11e8-99fe-fea69aa4b21b.png">

After:

<img width="310" alt="screen shot 2018-05-28 at 11 34 22 am" src="https://user-images.githubusercontent.com/52581/40591784-7eb89526-626b-11e8-9a8e-3d5ec86bdd07.png">

**Comment by DrewAPicture** [comment]
I'd rather you do it, yes. The styles might have something to do with
having reverted the markup changes in HTML->amount().

>

**Comment by amdrew** [comment]
👍 

While I'm in there I'm going to left-align the price.

Before

<img width="863" alt="screen shot 2018-05-31 at 11 27 33 am" src="https://user-images.githubusercontent.com/52581/40752931-da0e4b12-64c5-11e8-9736-f8b381b9395b.png">

After

<img width="869" alt="screen shot 2018-05-31 at 11 27 58 am" src="https://user-images.githubusercontent.com/52581/40752942-e4eaf6ac-64c5-11e8-8672-b7282b1b33f3.png">

**Comment by amdrew** [comment]
Mobile:

<img width="387" alt="screen shot 2018-05-31 at 11 57 12 am" src="https://user-images.githubusercontent.com/52581/40753739-ca2d99b0-64c9-11e8-9551-969b23f06223.png">

___

At desktop they move onto 1 line, with the `Remove` in its old position which matches the file fields further below for consistency. I may try at some point aligning the `Remove` to the far right in both sections (price and files) to see how it looks.

<img width="775" alt="screen shot 2018-05-31 at 11 57 01 am" src="https://user-images.githubusercontent.com/52581/40753748-d6588826-64c9-11e8-9f88-90c0ec189d36.png">

**Comment by DrewAPicture** [comment]
Left aligning the price just seems weird to me for some reason, which is
why it was right aligned before. I think it's because it's harder to scan
across multiple repeatable rows with the currency symbol on the front --
it's more difficult to separate the symbol from the value.

**Comment by amdrew** [comment]
@DrewAPicture If you force the validation errors to show with 1 field, and then add more rows, the validation errors show by default with each row created.

<img width="791" alt="screen shot 2018-05-31 at 12 26 26 pm" src="https://user-images.githubusercontent.com/52581/40754609-432bc234-64ce-11e8-92c7-d4c4ce05ab2e.png">

Is that something we can fix?

**Comment by amdrew** [comment]
Let's leave the price left-aligned (for now), going to look at it some more :)

**Comment by DrewAPicture** [comment]
No, not really. I noticed it too and even messed with it a little bit, but
because the error containers and formatting classes are dynamically
generated and injected by Parsley, hiding them when you clone a row is kind
of like whack-a-mole (hit or miss).

>

**Comment by amdrew** [comment]
Ok no problem, not a biggie

**Comment by amdrew** [comment]
Merged!
